### PR TITLE
Checks if response is a view before setting cookie

### PR DESCRIPTION
**Pull request by dawnerd**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,16 +11,19 @@ const defaults = {
 exports.register = (server, options, next) => {
   options = Object.assign({}, defaults, options);
 
-  server.ext('onPreHandler', (request, reply) => {
-    if (request.method !== 'get') {
+  server.ext('onPostHandler', (request, reply) => {
```
Comment (jgallen23) [inline index.js]: problem with doing onPostHandler, if we call the method on their first hit, where they have a referrer, but no cookie, it won't have anything.  If it's preHandler, it should, right?

Comment (dawnerd) [reply index.js]: If its preHandler we wont get if its a view or not.

Comment (dawnerd) [reply index.js]: Perhaps a better solution would be ignore the source and just block off paths

Comment (jgallen23) [reply index.js]: yeah, lets blacklist routes

File: README.md [hunk 2]
```diff
@@ -6,4 +6,5 @@ Hapi plugin to log referrers from search/ads/social/etc
  - `cookieName` - Name of the cookie to be set
  - `ttl` - cookie expiration, defaults to 30 days
  - `domains` - array of domains. Domains in the list will not have their referrer set
+ - `paths` - array of paths. Paths set here wont have cookies set. Note, applies to all paths matching the text. Example: `/public` would also block `/public/css/common.css`
```
Comment (jgallen23) [inline README.md]: this option is a little misleading.  Probably want to call it ignoredPaths

**Pull request merged**